practice-1/print_word_use_all.py

```diff
@@ -0,0 +1,19 @@
+"""Write a function named uses_all that takes a word and a string of required letters,
+and that returns True if the word uses all the required letters at least once."""
+def uses_all(word, required_letters):
+    for letter in required_letters:
+        if letter not in word:
+            return False
+    return True
+# print(uses_all())
```

Comment:
Should remove unused code. If the code comment is important, please add TODO comment.
Please check for all.